### PR TITLE
Added the version to tsdb stats api methods

### PR DESCRIPTION
**Pull request by xocasdashdash**
This sentence was missing from this endpoint but it is present in the rest of them. I think it might make sense to add a new rule to the PR template, just in the case of new API endpoints.

### CODE REVIEW

File: docs/querying/api.md [hunk 1]
```diff
@@ -871,6 +871,8 @@ $ curl http://localhost:9090/api/v1/status/buildinfo
 
 **NOTE**: The exact returned build properties may change without notice between Prometheus versions.
 
+*New in v2.15*
```
Comment (brian-brazil) [inline docs/querying/api.md]: This was added in 14: https://github.com/prometheus/prometheus/releases/tag/v2.14.0

Comment (xocasdashdash) [reply docs/querying/api.md]: Well, it is 404ing for me on this build version 2.14.0 (branch: HEAD, revision: edeb7a44cbf745f1d8be4ea6f215e79e651bfe19).

It seems the commit: https://github.com/prometheus/prometheus/commit/a85e7aac0e4b32bcc6fabbb6f4059842124bc05a was added to the 2.15.x release on the #6281 PR

Comment (brian-brazil) [reply docs/querying/api.md]: So it is, can you fix the changelog too?

Comment (xocasdashdash) [reply docs/querying/api.md]: Sure, I'm gonna go ahead and assume #6281 was a feature

**Comment by brian-brazil** [comment]
The entry needs to be removed from 2.14.0. Don't forget your DCO too.

**Comment by xocasdashdash** [comment]
> The entry needs to be removed from 2.14.0. Don't forget your DCO too.

I'm not seeing it in the 2.14.0 CHANGELOG , last PR mentioned there is #6279

**Comment by brian-brazil** [comment]
`* [FEATURE] API: `/api/v1/status/runtimeinfo` and `/api/v1/status/buildinfo` endpoints added for use by the React UI. #6243` is in 2.14

**Comment by xocasdashdash** [comment]
Mmmm that's a different API endpoint, this is tsdb related. Was this endpoint supposed to be added under that PR?

**Comment by brian-brazil** [comment]
/api/v1/status/buildinfo is the endpoint you're fixing the docs for.

**Comment by xocasdashdash** [comment]
Well, I think I was reading where to put this wrong. I based on how the tombstone clean up is done where the version it was added in is put in before the API description not at the end.

**Comment by brian-brazil** [comment]
These notes have always been at the end. It's additional information, not the first thing you need to know about an endpoint.

**Comment by xocasdashdash** [comment]
@brian-brazil OK! I now see it, I've updated the PR adding the correct version for the status/buildinfo (2.14.0) and put the tsdb status api version (2.15.0) on the correct place. 

I don't think the changelog needs changing in v2.14.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,14 +12,15 @@
 * [CHANGE] Discovery: Removed `prometheus_sd_kubernetes_cache_*` metrics. Additionally `prometheus_sd_kubernetes_workqueue_latency_seconds` and `prometheus_sd_kubernetes_workqueue_work_duration_seconds` metrics now show correct values in seconds. #6393
 * [CHANGE] Remote write: Changed `query` label on `prometheus_remote_storage_*` metrics to `remote_name` and `url`. #6043
 * [FEATURE] API: Added new endpoint for exposing per metric metadata `/metadata`. #6420 #6442
+* [FEATURE] API: Added new endpoint for exposing tsdb status `/status/tsdb`. #6281
```
Comment (brian-brazil) [inline CHANGELOG.md]: Don't mention it twice, just the enhancement here I guess.

Comment (xocasdashdash) [reply CHANGELOG.md]: Fixed.

**Comment by brian-brazil** [comment]
Thanks! I'll fix the changelog on the github release now.

**Pull request merged**